pkg/lang/ir/conda.go

```diff
@@ -27,6 +27,7 @@ import (
 const (
 	condarc             = "/home/envd/.condarc"
 	condaVersionDefault = "py39_4.11.0"
+	condaTempFile       = "/tmp/envd_conda_env.yaml"
```

Comment:
A new temp yaml file is created from rewriting the `name` field of user specification